tests/bluetooth/host/buf/bt_buf_get_evt/src/main.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+
+void test_main(void)
+{
+	uint32_t state;
+
+	ztest_run_registered_test_suites(&state);
```

Comment:
I show this funciton on the deprecated list. How is the Zephyr project treating deprecation in the test suites?

Reply:
This one, although it can be found under deprecated ZTest, was used in the `Test Framwork Getting started guide` version 3.1.0 and I just followed that one as it was the latest by that time.

Also, new APIs weren't available previously because the normal CMake build process wasn't available for unit test projects until the one who is now blocking my PR did some modifications.

I preferred not move to the new APIs because it would require some effort while I didn't receive any feedback from Nodic team regarding this point.

I may switch to the new APIs for the next UT module.